src/test/java/test/helm/Helm.java

```diff
@@ -0,0 +1,86 @@
+package test.helm;
+
+import io.vavr.collection.HashMap;
+import org.junit.jupiter.api.TestInfo;
+import test.model.KubeResources;
+import test.model.Product;
+
+import java.io.IOException;
+import java.lang.reflect.Method;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * A wrapper around the helm CLI tool, which we use for rendering a chart's templates and capturing the output.
+ */
+public final class Helm {
```

Comment:
The contents of this class were mostly extracted from the original unit test, but now it's a bit more general and is used by all the new tests.